spatialprofilingtoolbox/db/fractions_transcriber.py

```diff
@@ -3,8 +3,7 @@
 import pandas as pd
 
 from spatialprofilingtoolbox.db.database_connection import DatabaseConnectionMaker
```

Comment:
`create_data_analysis_study` and `importance_score_transcriber` have been changed to use psycopg2 connections directly, but not `fractions_transcriber`